src/main/kotlin/com/example/sushiordersystem/mapper/CustomerMapper.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.sushiordersystem.mapper
+
+import org.apache.ibatis.annotations.Insert
+import org.apache.ibatis.annotations.Mapper
+import org.apache.ibatis.annotations.Select
+import org.springframework.stereotype.Component
+import java.time.Instant
+
+data class CustomerEntity(
+        val id: String,
+        val tableId: String,
+        val checkedInAt: Instant
+)
+
+@Mapper
+@Component
+interface CustomerMapper {
+
+    @Select("""
+        SELECT EXISTS (
+            SELECT 1
+            FROM customers
+            WHERE table_id = #{tableId}
+        )
```

Comment:
```suggestion
        SELECT EXISTS (
            SELECT 1
            FROM tables
            WHERE id = #{tableId}
        )
```

のような気がする？
認識違ってたらすみません :pray:
（もしそうだったらTableMapperを作った方がいいと思います！）